pkg/doc/verifiable/credential.go

```diff
@@ -454,88 +438,115 @@ func decodeCredentialSchema(data []byte) ([]TypedID, error) {
 
 // NewCredential creates an instance of Verifiable Credential by reading a JSON document from bytes.
 // It also applies miscellaneous options like custom decoders or settings of schema validation.
-func NewCredential(vcData []byte, opts ...CredentialOpt) (*Credential, error) {
-	// Apply options
-	crOpts := defaultCredentialOpts()
-
-	for _, opt := range opts {
-		opt(crOpts)
-	}
+// It returns decoded Credential and its marshalled JSON. They can be used for extensions
```

Comment:
It isn't completely obvious what the return values represent. I get that there is a Credential and marshalled JSON. It's not so clear in the comment how that differs from the input? We should make this text very clear as it is the documentation for the package.

Reply:
Done! Could you check the new description, please?